Makefile

```diff
@@ -213,6 +215,8 @@ test/e2e/multitenant-rhoam/prow: export INSTALLATION_PREFIX := sandbox-rhoam
 test/e2e/multitenant-rhoam/prow: export IDENTITY_PROVIDER_NAME ?= DevSandbox
```

Comment:
hey @makslion I think we can remove this line as its exported already from env/mutlitenant